test/c/microstrain/CMakeLists.txt

```diff
@@ -1,11 +1,14 @@
-
-if(MICROSTRAIN_ENABLE_LOGGING)
-
+if(MIP_LOGGING_ENABLED)
     set(TEST_STRING_SOURCES
         "${CMAKE_CURRENT_LIST_DIR}/testutil_strings.h"
         "${CMAKE_CURRENT_LIST_DIR}/testutil_strings.c"
     )
 
+    # TODO: Replace with valid check for IO support
+    set(MICROSTRAIN_TEST_COMPILER_DEFINITIONS
+        "MICROSTRAIN_LOGGING_MAX_LEVEL=${MICROSTRAIN_LOGGING_MAX_LEVEL}"
+    )
+
```

Comment:
I think this should be enough for all the string-related tests, right? But also, it's going to be replaced with Alex's test stuff.